guides/common/modules/con_provisioning-methods-in-project.adoc

```diff
@@ -3,29 +3,49 @@
 [id="Provisioning-Methods-in-{ProjectNameID}_{context}"]
 = Provisioning methods in {ProjectName}
 
-With {ProjectName}, you can provision hosts by using the following methods.
+With {ProjectName}, you can provision hosts by using the following provisioning methods.
 
-Bare-metal hosts::
-{Project} provisions bare-metal hosts primarily by using PXE boot and MAC address identification.
-When provisioning bare-metal hosts with {Project}, you can do the following:
+Network based::
+You can use this method with hosts that are capable of PXE booting or UEFI HTTP booting.
 +
-* Create host entries and specify the MAC address of the physical host to provision.
-* Boot blank hosts to use the {Project} Discovery service, which creates a pool of hosts that are ready for provisioning.
-* Provision hosts with UEFI Secure Boot.
+You must configure the necessary network booting services, such as DHCP or TFTP.
+When you create a host entry, you select a PXE loader that can configure DHCP options for network booting if {SmartProxy} is set up to configure the DHCP service automatically.
++
+ifdef::satellite[]
+This method uses the Anaconda operating system installer and template-generated Kickstart script for installation automation downloaded from {Project}.
+endif::[]
 ifndef::satellite[]
-* Boot and provision hosts by using PXE-less methods.
+This method uses an operating system installer, such as Anaconda, Debian Installer, or YaST, and template-generated file with installation options downloaded from {Project}, such as Kickstart, Preseed, or AutoYaST.
 endif::[]
 
-Cloud providers::
-{Project} connects to private and public cloud providers to provision instances of hosts from images stored in the cloud environment.
-When provisioning from cloud with {Project}, you can do the following:
+Network based with PXE Discovery:::
+You can create host entries by discovering hosts on a network.
+The discovery process uses the Discovery image to boot the host and report its hardware details to {Project}.
+You can automate provisioning of discovered hosts by creating Discovery rules.
+
+:FeatureName: Kernel execution
+PXE-less Discovery with kexec::
+You can use this method with PXE-less hosts.
```

Comment:
Maybe instead of "PXE-less" something along the lines of "hosts that do not support network boot"?